src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -83,7 +79,7 @@ public void scaleUpAndDown() {
             });
 
             // Scaling down
-            pod = new ConfigureIndexRequestSpecPod().replicas(3);
+            pod = new ConfigureIndexRequestSpecPod().replicas(1);
```

Comment:
This didn't seem correct as it's validating for `replicas = 1` below in the `assertWithRetry`. I'm not sure how that was passing previously.